scripts/build-check

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+echo "Checks the build of all packages"
+
+cargo b -p spacewalk-primitives --all-features
+
+echo "Check all pallets"
+cargo b -p clients-info --all-features
+echo "excluding the feature testing-utils of pallet currency; that is for nightly build."
+cargo b -p currency --features std,testing-constants,try-runtime,runtime-benchmarks
```

Comment:
So the `currency` crate is the only reason we need to do this, correct?

Reply:
yes